messages/mms/send-mms-content.php

```diff
@@ -0,0 +1,24 @@
+<?php
+require_once __DIR__ . '../../config.php';
+require_once __DIR__ . '../../vendor/autoload.php';
+
+$keypair = new \Vonage\Client\Credentials\Keypair(
+    file_get_contents(VONAGE_APPLICATION_PRIVATE_KEY_PATH),
+    VONAGE_APPLICATION_ID
+);
+
+$client = new \Vonage\Client($keypair);
+
+$content = new \Vonage\Messages\MessageObjects\ContentObject(
```

Comment:
The [code snippet spec](https://github.com/Vonage/code-snippet-specs/blob/main/messages/mms/send-mms-content.md) uses an example of sending two content objects (an `image` and a `file` in the same message.